package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-spinners": "^0.13.7",
```

Comment:
I know npm provides the variety of libraries, so you might easily fall into installing a new library. But, I would not add a new library to only achieve this.

This is because currently there won't be places to use `react-spinners` besides here, so it would be over-killing.

https://tailwindcss.com/docs/animation
Since Tailwind already has a class of spinner, you can achieve the same spinner animation with just adding one vector image.

Reply:
No problem, I will work on that. Besides that, is the overall look alright?

Reply:
@DonUggioni Please, check my other comments as well.